lab-kyle/routes/note-route.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+const Note = require('../note-model/note');
+const response = require('../lib/response');
+const dataDir = __dirname + '/data';
+const Storage = require('../lib/storage');
+
+var storage = new Storage(dataDir);
+
+module.exports = function(router){
+  router
+  .post('/api/note', function(req, res){
+    if (req.body.content != undefined){
+      const note = new Note(req.body);
+      storage.setItem('notes', note).then(function(){
```

Comment:
This is okay, but it may make more sense to pass req.body.content to your note based on how your storage.setItem works (would avoid the ambiguity around things like the this.note.content.content assertion in your test).